docs/architecture/anti-entropy/hinted-handoff.rst

```diff
@@ -43,4 +43,65 @@ Hinted handoff is enabled and managed by these settings in :code:`scylla.yaml`:
 
 Storing of the hint can also fail. Enabling hinted handoff therefore does not eliminate the need for repair; a user must recurrently :doc:`run a full repair </operating-scylla/procedures/maintenance/repair/>` to ensure data consistency across the cluster nodes.
 
+Sync point HTTP API
+^^^^^^^^^^^^^^^^^^^
+
+ScyllaDB provides you with a means to wait until hints for given nodes have been replayed. It's realized by creating a so-called sync point that corresponds to the hints that have accumulated up to the time of its creation. It's always created for both regular hints and updates for materialized views. You can create and await sync points by sending HTTP requests to :code:`http://{API_ADDRESS}:{API_PORT}/hinted_handoff/sync_point/`, where :code:`{API_ADDRESS}` corresponds to the API IP address of the coordinator node which hints are supposed to be sent from.
+
+You can create a sync point by sending a POST request to :code:`/hinted_handoff/sync_point` with the following parameters:
+
+* :code:`target_hosts`: a list of the IP addresses of nodes towards which hints should be replayed; the IPs are divided by commas, e.g. :code:`127.0.0.2,127.0.0.3`. If no IP address is specified, ScyllaDB assumes the sync point should be created towards all live nodes in the cluster. This parameter is not required -- if it's not present, ScyllaDB will treat the request as if zero IP addresses were specified.
+
+ScyllaDB should return a JSON containing the ID of the created sync point as the response. In case of an error, a JSON reporting the issue will be returned instead.
+
+Examples:
+
+.. code-block::
+
+    $ curl -X POST http://127.0.0.1:10000/hinted_handoff/sync_point?target_hosts=127.0.0.2,127.0.0.3
+    "Ay4AAADgR8IYRSM14vPuN232DLKSAQAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAA/CncIC5VDTA=="
+
+    $ curl -X POST "http://127.0.0.1:10000/hinted_handoff/sync_point"
+    "Ay4AAADgR8IYRSM14vPuN232DLKSAQAMAAAAAAAAAAAAAAAMAAAAAAAAAAAAAAA/CncIC5VDTA=="
+
+    $ curl -X POST "http://127.0.0.1:10000/hinted_handoff/sync_point?target_hosts=invalid_host"
+    {"message": "Failed to parse host address invalid_host: invalid_host", "code": 400}
+
+You can await a previously created sync point by sending a GET request to :code:`/hinted_handoff/sync_point` with the following parameters:
+
+* :code:`id`: the ID of the sync point the user wants to await. This parameter is mandatory. Only one ID can be passed,
+* :code:`timeout`: an integer value specifying how many seconds ScyllaDB should wait for the hints corresponding to the sync point to be sent. If the value is negative, ScyllaDB will wait indefinitely long. This parameter is not mandatory; if it's not present, ScyllaDB will return the result immediately.
+
+The database should return a JSON containing one of the two values as the response:
+
+* :code:`"DONE"` -- if all of the hints corresponding to the passed sync point have already been sent,
```

Comment:
Nit: passing can also mean "go past something", and you could argue that "passing a sync point" could be interpreted in this second, alternative way. Let's avoid possible confusion and change "passed" to a word which doesn't have this second meaning, e.g. "provided".